pom.xml

```diff
@@ -306,6 +306,9 @@
           <groupId>org.benf</groupId>
           <artifactId>cfr</artifactId>
           <version>0.132.0</version>
+          <!-- since it is not on Maven central, we don't package it -->
+          <!-- if you need it, download it at http://www.benf.org/other/cfr/ and put the jar in the classpath yourself -->
+          <scope>provided</scope>
```

Comment:
It is not on maven central, but I've upload it on `maven.inria.fr`, so it is still publicly available. (So we could package it if we wanted it.)